docs/adguard-for-android/solving-problems/extending-restricted-settings.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Problems caused by extending restricted settings
+sidebar_position: 18
+---
+
+In an effort to improve user privacy, Google has expanded the restriction settings in Android 15. More settings, such as *App usage access*, now require an additional permission to be enabled. Since the *Firewall* and *Detect Samsung Pay* features require *App usage access* permissions to work properly, you may have experienced problems using them.  To resolve these issues, please follow the steps below:
+
+:::note
+
+AdGuard has a strict no-logging policy, which means that we do not collect your data or send it to third parties. We only ask for access to data without which the application cannot function.
+
+:::
+
+1. On your Android device, open the **Settings app**.
+1. Tap **Apps**.
+1. Tap **AdGuard**
+
+![AdGuard in Apps *mobile](https://cdn.adtidy.org/content/kb/ad_blocker/android/solving_problems/problems-caused-by-extending-restricted-settings/restricted1.png)
+
+If you can't find it, first tap See all apps or App info.
+
+1. Tap **More** (⁝) → *Allow restricted settings*.
+
+![Allow restricted settings *mobile](https://cdn.adtidy.org/content/kb/ad_blocker/android/solving_problems/problems-caused-by-extending-restricted-settings/restricted2.png)
+
+Follow the on-screen instructions.
+
+![Restricted settings allowed for AdGuard *mobile](https://cdn.adtidy.org/content/kb/ad_blocker/android/solving_problems/problems-caused-by-extending-restricted-settings/restricted3.png)
```

Comment:
It looks to me that by our logic this screenshot should come after the "All done" phrase, and not before it.